components/expression/expression-panel.html

```diff
@@ -48,4 +60,26 @@
       </tbody>
     </table>
   </div>
+  <div id="cell-info" class="panel-section box" style="display: none;">
+  </div>
+  <div id="cell-info-template" style="display: none;">
+    <button type="button" class="close">&times;</button>
+    <div id="gene">
+      <label>Gene</label>
+      <span></span>
+    </div>
+    <div id="condition">
+      <label>Condition</label>
+      <span></span>
+    </div>
+  </div>
+  <div id="path-info" class="panel-section box" style="display: none;">
```

Comment:
What is "path-info"? If this is for profiles, then probably you should name it "profile-info".